libs/shared/data/test/src/atoms.data.ts

```
@@ -36,7 +36,7 @@ export const createAtomsApiData = (
 ): Array<IInterfaceTypeDTO> =>
   atomsData.map((atom) => ({
     fields: [],
-    id: atom.api.id,
+    id: atom.api?.id ?? '',
```

since is required here, we should throw an error in the map callback if the id doesn't exist

---

Updated [here](https://github.com/codelab-app/platform/pull/2889/commits/6922102e6af31c4a2f0c5066c4ebe479ccdc906f).